package.json

```diff
@@ -14,12 +14,15 @@
   },
   "lint-staged": {
     "*": [
-      "prettier --write ."
+      "prettier --check . --write"
```

Comment:
I was getting a pre-commit hook failed error. Supposedly prettier is not checking for `.prettierignore` file which is resolved using the `--check` flag.
```
× prettier --write:
[error] No parser could be inferred for file: foo
```